_posts/2024-05-04-openloco-v24.05.md

```diff
@@ -0,0 +1,165 @@
+---
+title:      "OpenLoco version 24.05"
+tagline:    "Release blog post"
+author:     "Aaron van Geffen"
+date:       2024-05-04 13:00:00 +0200
+layout:     single
+classes:    wide
+categories: news
+---
+
+OpenLoco v24.05 is out! We've made so much _progress_ this month, so we're very excited to share this new release with you all.
+Let's dig into the details in this dev log. For a summary of changes, please find the [changelog]() on GitHub.
+
+## Progress bars are back, and better than ever! (#2411, #2440)
+
+One of the charming features of the original Locomotion game was its use of a train for a progress bar.
+This has been missing in OpenLoco for a while, but we've brought these progress bars back in this release!
+But why were they removed in the first place?
+
+When we reimplemented loading and saving games in C++, we were suprised by how much faster our implementation was,
+such that we elected progress bars were unnecessary. However, loading objects can still take a couple of seconds,
+especially on saves that pack a bunch.
+
+In addition to progress bars for loading and saving games, the game would use 'native' Windows progress bars
+before the game is launched, e.g. to indicate it was checking and indexing its object files. We never got around to
```

Comment:
I did actually do this but it wasn't cross platform and after talking to a few people there was no good cross platform solution so i never continued

Reply:
Ah, I thought that was for OpenRCT2! Would you like me to include a sentence for it?

Reply:
Nah it's fine